GVFS/GVFS.Common/GVFSLock.cs

```diff
@@ -319,7 +331,11 @@ public void AddStatsToTelemetry(EventMetadata metadata)
                 metadata.Add("DurationMS", this.lockAcquiredTime.ElapsedMilliseconds);
                 metadata.Add("LockHeldExternallyMS", this.lockHeldExternallyTimeMs);
                 metadata.Add("ParseGitIndexMS", this.parseGitIndexTimeMs);
+                metadata.Add("UpdatePlaceholdersTotalMS", this.placeholderTotalUpdateTimeMs);
                 metadata.Add("UpdatePlaceholdersMS", this.placeholderUpdateTimeMs);
+                metadata.Add("DeleteEmptyFolderPlaceholdersMS", this.placeholderEmptyFolderDeleteTimeMs);

```

Comment:
On Mac this time will also include re-expanding folders.  Should the name reflect that?

Reply:
I'm hesitant to name it based on something that only happens on one platform.  Maybe it should be named `CleanupFolderPlaceholdersMS` to give it a general name that would indicate multiple things could have happened?

Reply:
I think that's a good idea.  

Even sticking with the "Update" theme (e.g. `UpdateFolderPlaceholdersMS`) might be accurate enough because this is the code that actually processes them (as far as I'm aware).

(e.g. `UpdatePlaceholdersMS` includes deleting file placeholders)

Any general name works for me.